purchase_order_receipt_report/reports/purchase_order_receipt_templates.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+<template id="report_purchase_order_receipt_document">
+    <t t-call="web.external_layout">
+        <t t-set="o" t-value="o.with_context(lang=o.partner_id.lang)" />
+        <div style="position: absolute; top: 0px; right: 30px;">
+            <strong>Year</strong>&#160;&#160;&#160;&#160;
+            <strong>Month</strong>&#160;&#160;&#160;&#160;
+            <strong>Day</strong>
+        </div>
+        <t t-set="address">
+            <div
+                    t-field="o.partner_id"
+                    t-options='{"widget": "contact", "fields": ["address", "name", "phone", "vat"], "no_marker": True, "phone_icons": True}'
+                />
+        </t>
+        <t t-if="o.dest_address_id">
+            <t t-set="information_block">
+                <strong>Shipping address:</strong>
+                <div t-if="o.dest_address_id">
+                    <div
+                            t-field="o.dest_address_id"
+                            t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True, "phone_icons": True}'
+                            name="purchase_shipping_address"
+                        />
+                </div>
+
+            </t>
+        </t>
+        <div class="page">
+            <div class="oe_structure" />
+            <div class="mt-4">
+                <h2
+                        t-if="o.state in ['draft', 'sent', 'to approve']"
+                    >Quotation Receipt #<span t-field="o.name" /></h2>
+                <h2 t-if="o.state in ['purchase', 'done']">Receipt #<span
+                            t-field="o.name"
+                        /></h2>
+                <h2 t-if="o.state == 'cancel'">Cancelled Receipt #<span
+                            t-field="o.name"
+                        /></h2>
+            </div>
+            <div id="informations" class="row mt-4 mb32">
+                <div t-if="o.user_id" class="col-3 bm-2">
+                    <strong>Purchase Representative:</strong>
+                    <p t-field="o.user_id" class="m-0" />
+                </div>
```

Comment:
@kanda999 Please confirm this.
```suggestion
```

Reply:
@AungKoKoLin1997 
We should remove this.